modules/atomic-widgets/assets/js/editor/div-block-view.js

```diff
@@ -81,6 +81,15 @@ const DivBlockView = BaseElementView.extend( {
 		}
 
 		this.$el.addClass( this.getClasses() );
+		if ( changed?.tag !== undefined && this._parent ) {
+			this.rerenderEntireView();
+		}
```

Comment:
Please add an empty line
I think that the actual check should be if the `this.tagName()` (which returns the the tag name to render) is different than the actual tag rendered in the DOM (this.el.tagName.toLowerCase())

Maybe create a "compare tags" function and use it?
```suggestion
		if ( changed?.tag !== undefined && this._parent ) {
			this.rerenderEntireView();
		}
```

Reply:
compareActualAndExpectedTag
verifyTagMatch
Suggestions by GPT